adhocteam/Head-Start-TTADP
# fix bug when saving with no AR type selected

## Description of change
Due to recent changes, if a user attempts to save a report with first selecting recipient type, they will encounter an error. This is a fix for that.

## How to test
Start a new report. Jump to "Next Steps" without saving. The report should save.

## Issue(s)


## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
